index.js

```diff
@@ -0,0 +1,69 @@
+const core = require('@actions/core');
+const exec = require('@actions/exec');
+const common = require('@zaproxy/actions-common-scans');
+const _ = require('lodash');
+
+// Default file names
+let jsonReportName = 'report_json.json';
+let mdReportName = 'report_md.md';
+let htmlReportName = 'report_html.html';
+
+async function run() {
+
+    try {
+        let workspace = process.env.GITHUB_WORKSPACE;
+        let currentRunnerID = process.env.GITHUB_RUN_ID;
+        let repoName = process.env.GITHUB_REPOSITORY;
+        let token = core.getInput('token');
+        let docker_name = core.getInput('docker_name');
+        let target = core.getInput('target');
+        let format = core.getInput('format');
+        let rulesFileLocation = core.getInput('rules_file_name');
+        let cmdOptions = core.getInput('cmd_options');
+        let issueTitle = core.getInput('issue_title');
+        let failAction = core.getInput('fail_action');
+        let allowIssueWriting = core.getInput('allow_issue_writing');
```

Comment:
If you are using action common version `0.2.0`, then please do convert this to [boolean](https://github.com/zaproxy/action-baseline/pull/59/commits/a9dcad2ca65f739db41f88210c69cceabb2a4c34)

Reply:
Good catch, I thought it was already doing that. Will push that change.

Reply:
Done.